data/css/news.scss

```diff
@@ -118,3 +118,8 @@ $support-font: 'Fira Sans' !default;
         }
     }
 }
+
+.WindowSimple scrollbar {
+    // margin-top is for being covered by Layout.TopMenu
+    margin-top: 70px;
```

Comment:
I think this wasn't tested against the default news, I will tweak it as part of the rework.

![screenshot from 2018-03-27 15-32-35](https://user-images.githubusercontent.com/481418/37990446-4e0d7d8c-31d4-11e8-979e-2fc61ec93708.png)